proto/gen/googleapis/aiplatform/apiv1/aiplatformpb/dataset_service_aiptest.pb.go

```diff
@@ -15,6 +15,123 @@ import (
 	testing "testing"
 )
 
+// DatasetServiceTestSuiteConfigProvider is the interface to implement to decide which resources
+// that should be tested and configured.
+type DatasetServiceTestSuiteConfigProvider interface {
+	AnnotationTestSuiteConfig(t *testing.T) *DatasetServiceAnnotationTestSuiteConfig
+	AnnotationSpecTestSuiteConfig(t *testing.T) *DatasetServiceAnnotationSpecTestSuiteConfig
+	DataItemTestSuiteConfig(t *testing.T) *DatasetServiceDataItemTestSuiteConfig
+	DatasetTestSuiteConfig(t *testing.T) *DatasetServiceDatasetTestSuiteConfig
+	DatasetVersionTestSuiteConfig(t *testing.T) *DatasetServiceDatasetVersionTestSuiteConfig
+	SavedQueryTestSuiteConfig(t *testing.T) *DatasetServiceSavedQueryTestSuiteConfig
+}
```

Comment:
IMO add comment that returning nil from these methods is valid, and that it means the test is skipped.